packages/openapi3/src/schema-emitter-3-0.ts

```diff
@@ -253,6 +279,8 @@ export class OpenAPI3SchemaEmitter extends OpenAPI3SchemaEmitterBase<OpenAPI3Sch
     }
 
     const isMerge = checkMerge(schemaMembers);
+    const stdScalars = getStdScalarNames(schemaMembers);
```

Comment:
im pretty sure we already have this logic to decide on how to render a scalar

Reply:
I cannot find the logic... I just collect std scalars names here, return `Set<string>` , doesn't render it. renamed it.

Reply:
I was refering to `getSchemaForStdScalars` what mattered here is the schema that you got, if its inline then its not a problem, if its a $ref you need to cary it over.